terraform/testcases/prometheus_sd_adot_operator_awsprw/otconfig.tpl

```diff
@@ -0,0 +1,31 @@
+receivers:
+      prometheus:
```

Comment:
Is this tab spacing correct?

Reply:
It does look off, but again I did not write these and are exactly what they are in the current repository